allensdk/test/brain_observatory/behavior/behavior_project_cache/test_behavior_project_metadata_writer.py

```diff
@@ -13,82 +11,36 @@
 from allensdk.test.brain_observatory.behavior.conftest import get_resources_dir
 
 
-def convert_strings_to_lists(df, is_session=True):
-    """Lists when inside dataframe and written using .to_csv
-    get written as string literals. Need to parse out lists"""
-    df.loc[df['driver_line'].notnull(), 'driver_line'] = \
-        df['driver_line'][df['driver_line'].notnull()] \
-        .apply(lambda x: literal_eval(x))
-
-    if is_session:
-        df.loc[df['ophys_experiment_id'].notnull(), 'ophys_experiment_id'] = \
-            df['ophys_experiment_id'][df['ophys_experiment_id'].notnull()] \
-            .apply(lambda x: literal_eval(x))
-        df.loc[df['ophys_container_id'].notnull(), 'ophys_container_id'] = \
-            df['ophys_container_id'][df['ophys_container_id'].notnull()] \
-            .apply(lambda x: literal_eval(x))
-
-
-def sort_df(df: pd.DataFrame, sort_col: str):
-    """Sorts df for comparison"""
-    df = df.sort_values(sort_col)\
-        .reset_index()\
-        .drop('index', axis=1)
-    df = df[df.columns.sort_values()]
-    return df
-
-
 @pytest.mark.requires_bamboo
-def test_metadata():
-    release_date = '2021-03-25'
-    with tempfile.TemporaryDirectory() as tmp_dir:
-        bpc = VisualBehaviorOphysProjectCache.from_lims(
-            data_release_date=release_date)
-        bpmw = BehaviorProjectMetadataWriter(
-            behavior_project_cache=bpc,
-            out_dir=tmp_dir,
-            project_name='visual-behavior-ophys',
-            data_release_date=release_date)
-        bpmw.write_metadata()
+def test_metadata(tmpdir):
+    """tests that the metadata writer can reproduce metadata files
 
-        expected_path = os.path.join(get_resources_dir(),
-                                     'project_metadata_writer',
-                                     'expected')
-        # test behavior
-        expected = pd.read_pickle(os.path.join(expected_path,
-                                               'behavior_session_table.pkl'))
-        expected = sort_df(df=expected, sort_col='behavior_session_id')
-        obtained = pd.read_csv(os.path.join(tmp_dir,
-                                            'behavior_session_table.csv'),
-                               dtype={'mouse_id': str},
-                               parse_dates=['date_of_acquisition'])
-        obtained = sort_df(df=obtained, sort_col='behavior_session_id')
-        convert_strings_to_lists(df=obtained)
-        pd.testing.assert_frame_equal(expected,
-                                      obtained)
+    Notes
+    -----
+    this test breaks if any `BehaviorNwb` or `BehaviorOphysNwb` in
```

Comment:
This makes for kind of a worthless test. Maybe it might pick stuff up if something changes on the LIMS side of things, but I'm kind of in favor of just removing it.

Part of the problem is that I'm not even sure what the spirit of the test should be...

Reply:
I agree, it is a canary.
It will fail:
* if the data changes: I don't think the purpose of AllenSDK testing is to monitor changing LIMS data. This is the kind of thing we are trying to get away from.
* If the metadata values/columns change: presumably, this would be an intentional change.

I think we have a responsibility when undertaking something like data release to perform validation outside of AllenSDK tests that should capture these use cases.

@aamster is the originator of the test, but, since he is not available to comment and time is short, I say we go ahead and remove it.